Components/ChatInsightsHeader.tsx

```diff
@@ -14,8 +16,7 @@ interface ChatInsightsHeaderProps {
 const ChatInsightsHeader: React.FC<ChatInsightsHeaderProps> = ({ chatInsights }) => {
   const { state } = useUserContext();
 
-  // Determine whether to use web or phone data based on state.groupType
-  const isInternalAIChat = state.groupType === "InternalAIChat";
+  const isInternalAIChat = isGroupInProdOrDev(state.currentGroup, GroupIds.Brock) || isGroupInProdOrDev(state.currentGroup, GroupIds.Caleb);
```

Comment:
I ain't gonna lie, I'm kinda confused by what this does. Aren't all groups either prod or dev?

Reply:
For example, this code checks to see if the currentGroup is Brock or Caleb